index.html

```diff
@@ -1,70 +1,218 @@
+            <img src="images/menu.png" alt="Menu">
+          </a>
+        </nav>
+      </div>
+    </header>
   
-  <div class="main">
-
-    <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+    <div class="main" id="bg-main">
+      <div class="content">
+        <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+        <p>
+          I'm a software developer! I can help you 
+          build a product, feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don't 
+          hesitate to contact me.
+        </p>
+      </div>
+      <div class="socials">
+        <h4>LET'S CONNECT</h4>
+        <ul class="icon-bar">
+          <li><a href="#">
+            <img src="images/twitter.png" alt="Twitter">
+          </a></li>
+          <li><a href="#">
+            <img src="images/linkedin.png" alt="Linkedin">
+          </a></li>
+          <li><a href="#">
+            <img src="images/facebook.png" alt="Facebook">
+          </a></li>
+          <li><a href="#">
+            <img src="images/github.png" alt="Github">
+          </a></li>
+          <li><a href="#">
+            <img src="images/instagram.png" alt="Facebook">
+          </a></li>
+        </ul>
+      </div>
+    </div> <!-- End of MAIN -->
     
-    <div class="content">
-    <p>
-      I'm a software developer! I can help you 
-      build a product, feature or website Look 
-      through some of my work and 
-      experience! If you like what you see and 
-      have a project you need coded, don't 
-      hesitate to contact me.
-      </p>
+    <div class="works"> <!-- WORKS -->
+      <div class="bg-card">
+        <img src="images/tonic-1.png" alt="Tonic project seakpeak">
+        <h1>Tonic</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-  </div>
+      <div class="bg-card">
+        <img src="images/mps-1.png" alt="MPS project seakpeak">
+        <h1>Multi-Post 
+          <br>Stories</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
```

Comment:
- [ ] In reference to the project requirements, 👇🏻 It would be nice if you aligned the project elements with `flexbox`.

<img width="332" alt="juy" src="https://user-images.githubusercontent.com/4247740/144695583-2e566818-40dd-4924-8b44-c2c035ee600a.PNG">